openquake/hazardlib/tests/gsim/check_gsim.py

```diff
@@ -296,10 +297,9 @@ def _parse_csv_line(headers, values, req_site_params):
             value = float(value)
             if param == 'arias':  # ugly legacy corner case
                 param = 'ia'
-            imtclass = registry.get(param.upper(), None)
-            if imtclass:
-                imt = imtclass()
-            else:  # assume the IMT is a Spectral Acceleration
+            try:    # The title of the column should be IMT(args)
```

Comment:
With this we now can read any IMT in the test tables

Reply:
Good solution!